### PR TITLE
Private messaging

### LINKED ISSUE #142: Add real private messaging
Yes, you could add a private chat room with a certain user and chat like that but It takes up space in the list and we could add it and make it fit in a separate area.

### PR DESCRIPTION
**Pull request by 01000011-00110111**
Its ready.

resolves #142

### CODE REVIEW

**Comment by 01000011-00110111** [comment]
Its done but I think it would be nice to add an indicator to tell  if someone sent something

**Comment by 01000011-00110111** [comment]
I'm not sure if its fully ready for merge it cant use any commands and we to get the commands branch merged first so we can get commands working

**Comment by 01000011-00110111** [comment]
if you want we can turn back to a draft

**Comment by CastyiGlitchxz** [comment]
Everything seems fine to me.

**Review by CastyiGlitchxz — DISMISSED** [review]
Everything seems fine to me.

**Comment by cserver45** [comment]
you may want to confirm that what I did on `database.py` doesn't cause issues elsewhere (did the merge conflict resolution)

**Review by CastyiGlitchxz — DISMISSED** [review]
So far everything seems fine.

File: static/chat.js [hunk 1]
```diff
@@ -112,7 +112,7 @@ socket.on("private_data", (data) => {
     // update the url when the room is changed.
     // window.history.replaceState({"pageTitle": `${data['name']} - Chat`}, "", `/chat/${data['name']}`);
     // roomname = document.getElementById("RoomDisplay").innerHTML = '/'+data['name'];
-    window.history.replaceState({"pageTitle": `Private Chat`}, "", `/chat/${data['name']}`);
+    window.history.replaceState({"pageTitle": `Private Chat`}, "", `/Private`);
```
Comment (cserver45) [inline static/chat.js:115]: well, if your going to do this, might aswell include the other person's displayname in the uri (like what we do with rooms)

**Review by cserver45 — APPROVED** [review]
Looks good to me, approved.

**Pull request merged**